main.ml

```diff
@@ -528,7 +528,7 @@ let vdi_of_volume x =
     persistent = true;
   }
 
-let choose_datapath ?(persistent = true) response =
```

Comment:
Why the empty default? It might be better to be explicit about the domain always.

Reply:
fixed